src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/SecurityExtensionTest.php

```diff
@@ -415,6 +415,31 @@ public function testSwitchUserWithSeveralDefinedProvidersButNoFirewallRootProvid
         $this->assertEquals(new Reference('security.user.provider.concrete.second'), $container->getDefinition('security.authentication.switchuser_listener.foobar')->getArgument(1));
     }
 
+    public function testInvalidAccessControlWithEmptyRow()
+    {
+        $this->expectException(InvalidConfigurationException::class);
+        $this->expectExceptionMessage('There is an empty access control attributes set.');
```

Comment:
Those to lines should be moved downwards, right above the line that we expect to trigger the exception.

Reply:
Done!